pkg/rancher-desktop/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "rancher-desktop",
+  "description": "Rancher Desktop",
+  "private": false,
+  "devDependencies": {
+    "@vue/cli-plugin-babel": "~4.5.0",
+    "@vue/cli-service": "~4.5.0",
+    "@vue/cli-plugin-typescript": "^4.5.15"
+  },
+  "browserslist": [
+    "> 1%",
+    "last 2 versions",
+    "not dead"
+  ],
+  "icon": "https://raw.githubusercontent.com/rancher/ui-plugin-examples/main/pkg/clock/clock.svg",
```

Comment:
Wait what

Reply:
Some testing and troubleshooting and stayed around for longer than it deserved.. updated to be the Rancher Desktop icon from our icons library.